symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/AuthSession.java

```diff
@@ -0,0 +1,26 @@
+package com.symphony.bdk.core.auth;
+
+import com.symphony.bdk.core.auth.exception.AuthenticationException;
+
+/**
+ *
+ */
+public interface AuthSession {
+
+  /**
+   *
+   * @return
+   */
+  String getSessionToken();
```

Comment:
do we need to expose the complexity of the different type of tokens to the user?
ideally if I could just use a service as a bot or as a user (obo) like:

BdkConfig config = BdkConfig.load(...)
SymphonyBdk bdk = new SymphonyBdk(config)
bdk.messages().send(....) // do actions as bot by default
bdk.messages().send(User.username("bob"), ....) // obo is possible too
or maybe bdk.messages(User.username("bob")).send( ....) // obo is possible too

Reply:
That's a great idea! I'll introduce it here, but will complete it through the EPIC `APP-2806` (for services abstraction)